save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/HistoryView.kt

```diff
@@ -166,6 +166,16 @@ class HistoryView : AbstractView<HistoryProps, HistoryViewState>(false) {
                         }
                     }
                 }
+                column("contestName", "Participating in contest", { contestName ?: "N/A" }) { cellProps ->
+                    Fragment.create {
+                        td {
+                            a {
+                                href = getHrefToExecution(cellProps.row.original.id, cellProps.row.original.status, null)
```

Comment:
Seems confusing that clicking on contest name leads to execution view. Maybe, add a symbol like [this](https://fontawesome.com/icons/arrow-up-right-from-square?s=solid&f=classic) and change link's `href` to contest view for corresponding contest?

Reply:
Initially I did this like for all other columns, however yes, in this case it's better to redirect to the contest itself
Fixed